improver_tests/utilities/test_compare.py

```diff
@@ -242,12 +244,72 @@ def message_collector(message):
 
     # Check changing attribute type from int to float
     actual_ds.setncattr("whole_number", 4.0)
-    compare.compare_attributes("root", actual_ds, expected_ds, message_collector)
+    compare.compare_attributes("root", actual_ds, expected_ds, None, message_collector)
     assert len(messages_reported) == 1
     for part in ("int", "float", "whole_number"):
         assert part in messages_reported[0]
 
 
+def test_ignore_netcdf_attrs(dummy_nc):
+    """Check that changes to ignored attributes are ignored."""
+    actual_nc, expected_nc = dummy_nc
+    expected_ds = nc.Dataset(expected_nc, mode="r")
+    actual_ds = nc.Dataset(actual_nc, mode="a")
+
+    messages_reported = []
+
+    def message_collector(message):
+        messages_reported.append(message)
+
+    # Check modifying a simple attribute
+    actual_ds.setncattr("float_number", 3.2)
+    compare.compare_attributes(
+        "root", actual_ds, expected_ds, ["float_number"], message_collector,
+    )
+    assert len(messages_reported) == 0
+
+    # Reset attribute back to original value
+    actual_ds.setncattr("float_number", 1.5)
+    messages_reported = []
+
+    # Check modifying an array attribute
```

Comment:
Is this test unique?  Is it needed in addition to the above?

Reply:
It is distinct only in the sense that the attribute value is more complex (an array). It should be that it is ignored based on the name, so the value shouldn't matter. As such I will remove it.